keyring_fallback.go

```diff
@@ -0,0 +1,23 @@
+package keyring
+
+import (
+	"errors"
+	"runtime"
+)
+
+// All of the following method errors out on unsupported platforms
+var errUnsupportedPlatform = errors.New("Unsupported platform: " + runtime.GOOS)
```

Comment:
It would be helpful if this was exporter so users could check for this specific error.

Reply:
Would you mind creating a PR for this? :)

Reply:
Sure, #38.